pkg/bufferdecoder/eventsreader.go

```diff
@@ -211,7 +211,7 @@ func GetParamType(paramType string) ArgType {
 		return credT
 	case "umode_t":
 		return u16T
-	case "unsigned long[]":
+	case "unsigned long[]", "HookedFunctionData[]":
```

Comment:
Why can't you simply stay with "unsigned long[]" for your case ? I dont think we need to create another type here (as we're sticking with basic types unless we need to differentiate them somewhere else, not your case).

Reply:
I agree, but I think we have no other choice because in the event defections map this event argument is `HookedFunctionData[]`  which is because after the process of the event it casts its argument to be HookedFunctionData.

That is done because we submit to bpf array of addresses that each one is a function address that we want to check, and only after the check (if it hooked or not), I assign it to the HookedFunctionData struct with the name of the symbol of the address and the module owner